test/initialize.test.ts

```diff
@@ -10,6 +10,9 @@ vi.mocked(utils).computeResourceLimits = vi
   .mockReturnValue({ memFlag: '', cpuFlag: '' });
 vi.mock('../src/runUtils', () => ({
   getFilesDir: vi.fn().mockReturnValue('/mock/files/dir'),
+  getMountFlag: vi
```

Comment:
Can we please add unit tests of cases where there is no persistent storage, therefore the docker container doesn't have access to a persistent folder to read/write? 

Also, we can we update the README to make it clear that the FILES_DIR now is optional?